tests/wakunode_rest/test_rest_admin.nim

```diff
@@ -0,0 +1,79 @@
+{.used.}
+
+import
+  std/sequtils,
+  stew/shims/net,
+  testutils/unittests,
+  presto, presto/client as presto_client,
+  libp2p/crypto/crypto
+
+import
+  ../../waku/waku_core,
+  ../../waku/waku_node,
+  ../../waku/node/peer_manager,
+  ../../waku/waku_api/rest/server,
+  ../../waku/waku_api/rest/client,
+  ../../waku/waku_api/rest/responses,
+  ../../waku/waku_api/rest/admin/types,
+  ../../waku/waku_api/rest/admin/handlers as admin_api,
+  ../../waku/waku_api/rest/admin/client as admin_api_client,
+  ../../waku/waku_relay,
+  ../testlib/wakucore,
+  ../testlib/wakunode,
+  ../testlib/testasync
+
+suite "Waku v2 Rest API - Admin":
+  var node1 {.threadvar.}: WakuNode
+  var node2 {.threadvar.}: WakuNode
+  var node3 {.threadvar.}: WakuNode
+  var peerInfo2 {.threadvar.}: RemotePeerInfo
+  var peerInfo3 {.threadvar.}: RemotePeerInfo
+  var restServer {.threadvar.}: RestServerRef
+  var client{.threadvar.}: RestClientRef
+
+  asyncSetup:
+    node1 = newTestWakuNode(generateSecp256k1Key(), ValidIpAddress.init("127.0.0.1"), Port(60600))
```

Comment:
Let's not hardcode ports unless necessary - can you please use `Port(0)` instead for waku nodes?

Reply:
Hmm, no, Port(0) really just means "let the OS decide an available port", so there should be no difference from the perspective of this test - and if it is, there is a bug somewhere:)

Let's keep the ports in, but let's take a look at it